docs/ref/middleware.txt

```diff
@@ -93,18 +93,30 @@ GZip middleware
 
 .. class:: GZipMiddleware
 
+.. versionchanged:: 4.2
+
+    Mitigation for the BREACH attack was added. See below for more information.
+
 .. warning::
 
-    Security researchers recently revealed that when compression techniques
+    Security researchers revealed that when compression techniques
     (including ``GZipMiddleware``) are used on a website, the site may become
-    exposed to a number of possible attacks. Before using ``GZipMiddleware`` on
-    your site, you should consider very carefully whether you are subject to
-    these attacks. If you're in *any* doubt about whether you're affected, you
-    should avoid using ``GZipMiddleware``. For more details, see the `the BREACH
-    paper (PDF)`_ and `breachattack.com`_.
+    exposed to a number of possible attacks.
+
+    Before using ``GZipMiddleware`` on your site, you should consider very
+    carefully whether you are subject to these attacks. If you're in *any* doubt
+    about whether you're affected, you should avoid using ``GZipMiddleware``.
+
+    To mitigate attacks, Django implements a technique called Heal The Breach
+    (HTB). It adds up to  100 bytes of random bytes to each response to make the
+    attacks less effective.
+
+    For more details, see the `the BREACH paper (PDF)`_, `breachattack.com`_ and
+    the `Heal The Breach (HTB) paper`_.
```

Comment:
```suggestion
    the `Heal The BREACH (HTB) paper`_.
```

Reply:
I made sure to uppercase BREACH and use "Heal The Breach (HTB)" consistently now. That's the way its written in the HTB paper :)